src/views/CreateProfile.vue

```diff
@@ -256,16 +256,16 @@
 </template>
 
 <script>
-  import HeaderComponent from '../components/layout/HeaderComponent.vue';
-  import LeftSectionComponent from '../components/layout/LeftSectionComponent.vue';
-  import RightSectionComponent from '../components/layout/RightSectionComponent.vue';
+  import Header from '../components/layout/Header.vue';
+  import LeftSection from '../components/layout/LeftSection.vue';
+  import RightSection from '../components/layout/RightSection.vue';
 
   export default {
     name: 'CreateProfile',
     components: {
-      'header-component': HeaderComponent,
-      'left-section': LeftSectionComponent,
-      'right-section': RightSectionComponent,
+      'custom-header': Header,
+      'left-section': LeftSection,
```

Comment:
the mapping is not needed for all if the name is the same

Reply:
There is a <header> html5 tag, so probably we have to leave it for the header.